packages-ts/starknet-gauntlet/src/wallet/ledgerWallet.ts

```diff
@@ -0,0 +1,94 @@
+import { IStarknetWallet } from './'
+import { SignerInterface, encode } from 'starknet'
+import { Abi, Invocation, InvocationsSignerDetails } from 'starknet/types'
+import { TypedData, getMessageHash } from 'starknet/utils/typedData'
+import { fromCallsToExecuteCalldataWithNonce } from 'starknet/utils/transaction'
+import { calculcateTransactionHash, getSelectorFromName } from 'starknet/utils/hash'
+import Stark from '@ledgerhq/hw-app-starknet'
+import { LedgerError } from '@ledgerhq/hw-app-starknet'
+import TransportNodeHid from '@ledgerhq/hw-transport-node-hid'
+
+// EIP-2645 path
+const PATH = "m/2645'/579218131'/0'/0'"
```

Comment:
```suggestion
const DEFAULT_PATH = "m/2645'/579218131'/0'/0'"
```

Reply:
@sdrug we should predefine the `application` here: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2645.md

> application - the 31 lowest bits of sha256 of the application name. Serve as a domain separator between different applications. In the context of DeversiFi in June 2020, it is the 31 lowest bits of sha256(starkexdvf) and the value would be 1393043894.

1. I suggest we use "chainlink" as the application string
2. We should check that every path (user provided) starts with the `m/2645'/579218131'/`